rust/foxglove/src/websocket.rs

```diff
@@ -881,7 +993,7 @@ impl Server {
         Ok(local_addr)
     }
 
-    pub async fn stop(&self) {
+    pub fn stop(&self) {
```

Comment:
Do you think we should release a patch version with some of the fixes & doc updates prior to the next breaking change?

Reply:
NM, this is not breaking. I see you also have #362 for breaking changes.